src/main/resources/db/changelog/changes/05-insert-role-admin.yaml

```diff
@@ -0,0 +1,28 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert-role-admin
+      author: alex
+      changes:
+        - insert:
+            tableName: users
+            columns:
+              - column: { name: user_name, value: "admin" }
+              - column: { name: password, value: "$2a$10$avTiBCqM1zD9i4MYRyTpiuGAd02On.AGnq7v9SKRrq0k39AAydVKu" }
```

Comment:
include it in plaintext as a comment if it's for testing purposes

Reply:
the admin needs to put roles